org-formation/700-aws-sso/_tasks.yaml

```diff
@@ -979,3 +983,73 @@ SsoCnbAdmin:
     instanceArn: !Ref instanceArn
     principalId: !Ref cnbAdminGroup
     permissionSetArn: !CopyValue [ !Sub '${resourcePrefix}-${appName}-admin-permission-set-arn' ]
+
+SsoScicompIncludeDccCollab:
+  Type: update-stacks
+  Template: https://raw.githubusercontent.com/Sage-Bionetworks/aws-infra/v0.2.11/templates/SSO/aws-sso.yaml
+  StackName: !Sub '${resourcePrefix}-${appName}-scicomp-include-dcc-collab'
+  StackDescription: 'SSO: role to access INCLUDE DCC project resources at CHOP'
+  DefaultOrganizationBindingRegion: !Ref primaryRegion
+  DefaultOrganizationBinding:
+    IncludeMasterAccount: true
+  OrganizationBindings:
+    TargetBinding:
+      Account: !Ref ScicompAccount
+  Parameters:
+    instanceArn: !Ref instanceArn
+    principalId: !Ref scicompIncludeDccCollabGroup
+    permissionSetName: 'IncludeDccCollab'
+    sessionDuration: 'PT12H'
+    # policy derived from https://docs.aws.amazon.com/vpc/latest/privatelink/vpc-endpoints-s3.html#vpc-endpoints-policies-s3
+    # Restricting access to buckets in a specific AWS account
+    inlinePolicy: >-
+      {
+        "Version": "2012-10-17",
+        "Statement": [
+          {
+            "Effect": "Deny",
```

Comment:
According to these [docs](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_evaluation-logic-cross-account.html), these `Deny` statements won't be sufficient to grant cross-account access. I think the role also needs explicit `Allow` statements for S3 actions.

Reply:
good catch.  I've corrected the policy.  It now provide S3 full access to this role with an explicit S3 deny on all accounts except for the CHOP accounts.